cli/wallet.go

```diff
@@ -101,7 +102,10 @@ var walletBalance = &cli.Command{
 		}
 
 		balance, err := api.WalletBalance(ctx, addr)
-		if err != nil {
+
+		if errors.Is(err, types.ErrActorNotFound) {
```

Comment:
errors.Is won't work on errors that cross the RPC boundary, we lose that extra information

Reply:
Aaah, right.